src/main/java/org/pesmypetcare/webservice/entity/WeightEntity.java

```diff
@@ -0,0 +1,8 @@
+package org.pesmypetcare.webservice.entity;
+
+import lombok.Data;
+
+@Data
+public class WeightEntity {
```

Comment:
Is it really necessary to have a class with just one attribute? Wouldn't it be better to use a variable instead?

Reply:
To solve, we are going to fix it in the next iteration.